integrations/gitlab/gitlab_integration/gitlab_service.py

```diff
@@ -615,6 +648,8 @@ def _parse_file_content(
         except json.JSONDecodeError:
             try:
                 documents = list(yaml.load_all(file.decode(), Loader=yaml.SafeLoader))
+                if not documents:
+                    return file.decode().decode("utf-8")
```

Comment:
what does this fixes? is it another bug discovered?

Reply:
I've tried loading markdown and got a really weird error from this line, that's why I added it here